docs/CHANGELOG.md

```diff
@@ -9,7 +9,7 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 The format of this change log follows the advice given at [Keep a CHANGELOG](http://keepachangelog.com).
 
 ## [Unreleased]
-No unreleased changes.
+- Updated composer.json to use version 2.9.8 of the `local_codechecker` plugin.
```

Comment:
For some reason I think I prefer to say:

    - Updated version of `moodlehq/moodle-local_codechecker` to v2.9.7

Note this is exclusively for `local_codechecker`, because we produce real versioned releases. For others like `local_ci` or `local_moodlecheck`, that don't have real releases, surely it's ok to say, simply:

    - Updated project dependencies (`local_ci`, `other_important_component`...)

I'm not strong on that, just sharing my opinion, because the "Updated composer.json to use..." doesn't sound great in my brain.

Opinions?

Reply:
It's ok for me.  So the rule would be to write "Updated version of..." when there are real versions, "Updated project dependencies..." otherwise, right?

Reply:
For me it makes sense, yep. Let's see if anybody chimes in with other idea and let's go that way on Monday, if that's ok for you.

Reply:
> Updated project dependencies (`local_ci`, `other_important_component`...)
I agree, we update dependencies.

Reply:
Thanks @kabalin,

so, with 3 of us agreeing, sold:

1) for versioned / key dependencies `Updated version of xxxx to yyy`
2) for all the non-versioned / general dependencies `Updated project dependencies (optional list of key ones)`